rtb/exchange/multibidder_communicator.hpp

```diff
@@ -27,21 +27,24 @@ namespace vanilla {
 
     template
     <  
+        typename DSL,
         typename Duration  = std::chrono::milliseconds, 
         typename DeliveryType = vanilla::messaging::broadcast
     > 
     class multibidder_communicator {
+    private:
+            using serialized_type = typename DSL::serialized_type;
     public:
         multibidder_communicator(uint16_t bidders_port, Duration response_timeout) :
             response_timeout(response_timeout) {
             communicator.outbound(bidders_port);
         }
 
-        template<typename T>
-        void process(const vanilla::VanillaRequest &vanilla_request, multibidder_collector<T> &collector) {
+        template<typename Request>
```

Comment:
I think we can use any collector here ?

```
 template<typename Request, typename Collector>
void process(const Request &request,  Collector &collector) {
```
I would also suggest removing depencies on very nested types 
DSL::serialized_type::data_type , and even further maybe in future replace
DSL::deserialized_type -> DSL::request_type
DSL::serialized_type -> DSL::response_type

Because I sometimes forget which one is which :-)